pages/login.vue

```diff
@@ -1,4 +1,7 @@
 <script setup>
+definePageMeta({
+  layout: "login",
+});
```

Comment:
I would suggest adding this at the end of script before </script>

Reply:
done